fix: include all auction claims when closing up an auction

sborrazas opened the pull request:
Requires re-sync

jyeshe commented on lib/ae_mdw/db/mutations/name_claim_mutation.ex:
```diff
@@ -120,33 +120,49 @@ defmodule AeMdw.Db.NameClaimMutation do
 
+              auction_claim = Model.auction_bid_claim(index: {plain_name, start_height, txi_idx})
+
+              m_auction_bid =
+                Model.auction_bid(
+                  index: plain_name,
+                  start_height: start_height,
+                  block_index_txi_idx: {block_index, txi_idx},
+                  expire_height: auction_end,
+                  owner: owner_pk
+                )
```
Not an operational issue but as the purpose here is an update, would use the original record and set only fields that needs to be updated to make it more clear what it is changing.

sborrazas replied:
Done

jyeshe reviewed (changes requested):
There are some points where it references `expired_height` instead of `started_height` for an `AuctionBidClaim`:
https://github.com/aeternity/ae_mdw/blob/aaae71a61301210ae7fcb3f16cab43b5ab03fcbe/lib/ae_mdw/auction_bids.ex#L154
https://github.com/aeternity/ae_mdw/blob/aaae71a61301210ae7fcb3f16cab43b5ab03fcbe/lib/ae_mdw/names.ex#L763-L768

We would be great to have the test suite covering this gap.

sborrazas commented:
> There are some points where it references `expired_height` instead of `started_height` for an `AuctionBidClaim`:
> 
> https://github.com/aeternity/ae_mdw/blob/aaae71a61301210ae7fcb3f16cab43b5ab03fcbe/lib/ae_mdw/auction_bids.ex#L154
> 
> 
> https://github.com/aeternity/ae_mdw/blob/aaae71a61301210ae7fcb3f16cab43b5ab03fcbe/lib/ae_mdw/names.ex#L763-L768
> 
> We would be great to have the test suite covering this gap.

Good find, fixed the /auctions endpoint and updated tests accordingly which started to fail. thanks
